src/interface.jl

```diff
@@ -83,33 +83,67 @@ Identity(::LieGroup{𝔽,O}) where {𝔽,O<:AbstractGroupOperation} = Identity{O
 Identity(::O) where {O<:AbstractGroupOperation} = Identity(O)
 Identity(::Type{O}) where {O<:AbstractGroupOperation} = Identity{O}()
 
+"""
+    AbstractLieGroupPoint <: ManifoldsBase.AbstractManifoldPoint end
+
+An abstract type for a point on a [`LieGroup`](@ref).
+While an points and tangent vectors are usually kept untyped for flexibility,
+it might be necessary to distinguish different types of points, for example
+
+* for complicated representations that require a struct
+@ semantic verification
+* when there exist different representations
```

Comment:
What does this `@` mean here?

Reply:
Ah that is just a typo and should be a `*` (both @ and * are the same key on the Norwegian Keyboard, * being shift+@).